app/src/ui/list.tsx

```diff
@@ -560,7 +594,9 @@ export class List extends React.Component<IListProps, IListState> {
 
     return (
       <Grid
+        aria-label={''}
```

Comment:
Are this and `role` intended to be empty strings? If so, we shouldn't need the wrapping curlies.